### PR TITLE
Previous Google Summer of Code projects with final reports

### PR DESCRIPTION
**Pull request by jorgensd**

### CODE REVIEW

File: gsoc.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- Igor Baratta (Universidade Federal de Minas Gerais)- [Interface to KaHIP partitioner](https://gist.github.com/IgorBaratta/038747f9aff44eb844c0913614bf0c03)
+
+## 2018
+
+- Fabian Löschner (RWTH Aachen University) - [Maximising performance on modern architectures with data-level parallelism](https://flgsoc18.wordpress.com/2018/05/13/excited-for-fenics-and-gsoc/)
+- Igor Baratta (Universidade Federal de Minas Gerais)- [Complex Number support in FEniCS](https://gist.github.com/IgorBaratta/c7ca5252834f2c70efe0d233a3acecb4)
+
+## 2017
+
+- Ivan Yashcuch (Aalto University) -[Develop assembly of finite element forms on quadrilateral and hexahedral meshes in FEniCS](https://nbviewer.org/gist/IvanYashchuk/48bff62b155e31f85de975b24b23e4cb/)
```
Comment (blechta) [inline gsoc.md]: The spelling of the last name looks dubious. The white space around the hyphen (there should be en dash, in fact) is also random around the file.

Comment (jorgensd) [inline gsoc.md]:
```suggestion
- Ivan Yashchuk (Aalto University) - [Develop assembly of finite element forms on quadrilateral and hexahedral meshes in FEniCS](https://nbviewer.org/gist/IvanYashchuk/48bff62b155e31f85de975b24b23e4cb/)
```

File: gsoc.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Google Summer of Code
+
+The FEniCS Project has participated in the [Google Summer of Code](https://summerofcode.withgoogle.com) (GSoC) through our fiscal sponsorship agreement with [NumFOCUS](https://numfocus.org/).
+
+## Previous projects
+
+## 2022
+
+- Michele Castriotta (Istituto Italiano di Tecnologia) - [Expanding FEniCSx electromagnetic demos](https://mikics.github.io/gsoc-jupyterbook/submission.html)
```
Comment (jorgensd) [inline gsoc.md]:
```suggestion
- Michele Castriotta (Istituto Italiano di Tecnologia): [Expanding FEniCSx electromagnetic demos](https://mikics.github.io/gsoc-jupyterbook/submission.html)
```

File: gsoc.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+## 2019
+
+- Abhinav Gupta (Indian Institute of Technology, Roorkee) - [FEniCS: The mesh workflow](https://computationalmechanics.in/fenics-the-mesh-workflow/)
```
Comment (jorgensd) [inline gsoc.md]:
```suggestion
- Abhinav Gupta (Indian Institute of Technology, Roorkee): [FEniCS: The mesh workflow](https://computationalmechanics.in/fenics-the-mesh-workflow/)
```

File: gsoc.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+- Igor Baratta (Universidade Federal de Minas Gerais)- [Interface to KaHIP partitioner](https://gist.github.com/IgorBaratta/038747f9aff44eb844c0913614bf0c03)
```
Comment (jorgensd) [inline gsoc.md]:
```suggestion
- Igor Baratta (Universidade Federal de Minas Gerais): [Interface to KaHIP partitioner](https://gist.github.com/IgorBaratta/038747f9aff44eb844c0913614bf0c03)
```

File: gsoc.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+## 2018
+
+- Fabian Löschner (RWTH Aachen University) - [Maximising performance on modern architectures with data-level parallelism](https://flgsoc18.wordpress.com/2018/05/13/excited-for-fenics-and-gsoc/)
```
Comment (jorgensd) [inline gsoc.md]:
```suggestion
- Fabian Löschner (RWTH Aachen University): [Maximising performance on modern architectures with data-level parallelism](https://flgsoc18.wordpress.com/2018/05/13/excited-for-fenics-and-gsoc/)
```

File: gsoc.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+- Igor Baratta (Universidade Federal de Minas Gerais)- [Complex Number support in FEniCS](https://gist.github.com/IgorBaratta/c7ca5252834f2c70efe0d233a3acecb4)
```
Comment (jorgensd) [inline gsoc.md]:
```suggestion
- Igor Baratta (Universidade Federal de Minas Gerais): [Complex Number support in FEniCS](https://gist.github.com/IgorBaratta/c7ca5252834f2c70efe0d233a3acecb4)
```

File: gsoc.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+## 2017
+
+- Ivan Yashchuk (Aalto University) - [Develop assembly of finite element forms on quadrilateral and hexahedral meshes in FEniCS](https://nbviewer.org/gist/IvanYashchuk/48bff62b155e31f85de975b24b23e4cb/)
```
Comment (jorgensd) [inline gsoc.md]:
```suggestion
- Ivan Yashchuk (Aalto University): [Develop assembly of finite element forms on quadrilateral and hexahedral meshes in FEniCS](https://nbviewer.org/gist/IvanYashchuk/48bff62b155e31f85de975b24b23e4cb/)
```

File: gsoc.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+- Michal Habera (Charles University, Prague) - [Develop XDMF format for visualisation and checkpointing](https://github.com/michalhabera/gsoc-summary/blob/master/README.md/)
```
Comment (jorgensd) [inline gsoc.md]:
```suggestion
- Michal Habera (Charles University, Prague): [Develop XDMF format for visualisation and checkpointing](https://github.com/michalhabera/gsoc-summary/blob/master/README.md/)
```

**Review by garth-wells — COMMENTED** [review]
I don't Gsoc belongs in the governance docs. Governance docs cover how we are organised and function, and should be kept as tight and focussed as possible.

**Comment by jorgensd** [comment]
> I don't Gsoc belongs in the governance docs. Governance docs cover how we are organised and function, and should be kept as tight and focussed as possible.

Where do we want to keep them? The web-page? I only put the PR in here after @jhale suggested it.

**Comment by jorgensd** [comment]
Closed in favor of: https://github.com/FEniCS/governance/pull/14

**Pull request closed without merging**